pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 norecursedirs = .git .tox *.egg-info build node_modules TOXENV benchmarks prototypes
-addopts = --strict -m "not slow and not dist and not flaky"
+addopts = --strict -m "not slow and not dist and not flaky" --benchmark-disable
```

Comment:
See comment above, this may need to be removed for more intuitive interactive use.